source/fundamentals/connection/tls.txt

```diff
@@ -152,6 +155,36 @@ please refer to the `JSSE Reference Guide
 
 .. _tls-disable-hostname-verification:
 
+Configure a Client-Specific Trust Store and Key Store
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can configure a client-specific trust store and key store using the
+``init()`` method of the ``SSLContext`` class.
+
+The following code snippet configures a client-specific trust store and key
+store:
+
+.. code-block:: java
+
+   String algorithm = "TLS";
+   try {
+      final SSLContext sslContext = SSLContext.getInstance(algorithm);
+      sslContext.init(<Your Key Managers>, <Your Trust Managers>, <A Secure Random Number Generator>);
+      MongoClientSettings settings = MongoClientSettings.builder()
+               .applyToSslSettings(builder -> {
+                     builder.enabled(true);
+                     builder.context(sslContext);
+               })
+               .build();
+      MongoClient client = MongoClients.create(settings);
+   }
+   catch (NoSuchAlgorithmException e) {
+      System.err.println(algorithm + " is not a valid message digest algorithm");
+   }
+
+For more information on the ``SSLContext`` class, see the official Oracle API
```

Comment:
The above example contains lots of code that does not convey much helpful information. Following are the things we need to tell the user:

1. you can configure driver-specific key store and trust-store (was already done verbally before the code sample);
2. you can use `SSLContext.init` to do this (was already done verbally before the code sample; all we need is to specify the method name and the link to its documentation as it was done in the old driver docs: [javax.net.ssl.SSLContext.init(KeyManager[] km, TrustManager[] tm, SecureRandom random](https://docs.oracle.com/en/java/javase/16/docs/api/java.base/javax/net/ssl/SSLContext.html#init(javax.net.ssl.KeyManager%5B%5D,javax.net.ssl.TrustManager%5B%5D,java.security.SecureRandom)));
3. you can then tell the driver to use your custom `SSLContext` as described in ["Customize TLS/SSL Configuration with an SSLContext"](https://docs.mongodb.com/drivers/java/sync/current/fundamentals/connection/tls/#customize-tls-ssl-configuration-with-an-sslcontext).

I.e., the whole part can be documented in three sentences of which two are already in this PR. I propose removing the code sample and replacing it with a sentence as described in item 3.

Reply:
Thank you for the note. Re-reading this section with your comments in mind, I can see how the code snippet does not provide much helpful information for readers. I have removed the code snippet, and linked to the "Customize TLS/SSL Configuration..." section as suggested. Here is the fix: https://github.com/mongodb/docs-java/pull/139/commits/c2803079ee4485e005c4627f6b3f3da68cfa05f9